tests/functional/i/invalid/invalid_name.py

```diff
@@ -72,5 +72,24 @@ def a_very_very_very_long_function_name_WithCamelCase_to_make_it_sad():  # Shoul
 
 
 class FooBar:
-    def __init__(self, fooBar) -> None: # [invalid-name]
+    def __init__(self, fooBar) -> None:  # [invalid-name]
+        self.foo_bar = fooBar
+
+    def func1(
+        self,
+        fooBar,  # [invalid-name]
+    ):
+        self.foo_bar = fooBar
+
+    # Test disable invalid-name
+    def test_disable1(self, fooBar):  # pylint: disable=invalid-name
+        self.foo_bar = fooBar
+
+    def test_disable2(
+        self,
+        fooBar,  # pylint: disable=invalid-name

```

Comment:
```suggestion
        bar,  # [unused-argument] 
        fooBar,  # pylint: disable=invalid-name
        barFoo,  # [invalid-name, unused-argument]
```

Reply:
I added another case: `test_disable_mixed`. Let me know if that is what you wanted to test here.

Reply:
Yes perfect :)